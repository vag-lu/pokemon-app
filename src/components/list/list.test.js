import { shallow, configure } from 'enzyme';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import List from '.';

configure({ adapter: new Adapter() });

describe('List', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<List/>);
        expect(wrapper).toMatchSnapshot();
    })
})