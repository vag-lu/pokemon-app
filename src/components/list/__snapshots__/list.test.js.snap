// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders correctly 1`] = `ShallowWrapper {}`;

exports[`PokemonCard renders correctly 1`] = `ShallowWrapper {}`;
